cmd/create/machinepool/machinepool.go

```diff
@@ -22,6 +22,12 @@ import (
 func addMachinePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r *rosa.Runtime) {
 	var err error
 
+	isVersionSet := cmd.Flags().Changed("version")
+	if isVersionSet {
```

Comment:
Do we need this validation for the `edit machine pool` command for classic ROSA as well?

Reply:
Right, "version" parameter should not be supported in classic rosa, so it should be added there, too.